docs/components/maintenance.rst

```diff
@@ -1,4 +1,66 @@
-Maintenance
-###########
+Extending maintenance cleanup
+#############################
 
+To hook into the ``mautic:maintenance:cleanup`` command, create a listening for the ``\Mautic\CoreBundle\CoreEvents::MAINTENANCE_CLEANUP_DATA`` event.
+The event listener should check if data should be deleted or a counted. Use ``$event->setStat($key, $affectedRows, $sql, $sqlParameters)`` to give feedback to the CLI command.
```

Comment:
Not sure what you mean by 'a counted' here?

Reply:
Yeah no problem at all - I didn't understand the English words or what was intended, I guess it meant to say 'or count the number of affected records' perhaps.